config/routes.rb

```diff
@@ -193,6 +194,14 @@
   end
 
   scope 'dev' do
+    post 'st_functional',    to: 'developer#st_mark_functional'
+    post 'st_broken',        to: 'developer#st_mark_broken'
+    post 'st_add_post',      to: 'developer#st_insert_post'
+    post 'st_add_post_sync', to: 'developer#st_insert_post_synchronous'
+    post 'st_add_range',     to: 'developer#st_insert_post_range'
+    get  'st_basic_search',  to: 'developer#st_basic_search_raw'
+    post 'st_sync',          to: 'developer#st_sync'
+    post 'st_async_sync',    to: 'developer#st_async_sync'
```

Comment:
All the dev routes (and search_st) should end up in the nav or at least linked from somewhere

Reply:
I have absolutely zero knowledge/experience in UI design or front end dev. I am not sure how to do this... Maybe make a big "Suffix tree control panel"?

Reply:
You should be able to add them in /app/views/layouts/application.html.erb, that's where the navbar gets built.